### DISCUSSION TITLE
breaking a long line

### DISCUSSION THREAD

**Discussion by fauxbytes** [motdotla/dotenv#729] [Q&A] [answered]
Hi.
Is there a way to declare a variable over multiple lines, then have dotenv concatenate it back to a single line?
sth like:
```
VAR="""this is a v
ery looooooong value, whi
ch should be evaluated to a single, non-
broken line"""
```
and when evaluated in code:
```
let v=process.env.VAR;
//v="this is a very looooooong value and which should be evaluated to a single, non-broken line"
```

**Comment by motdotla** [comment, marked as answer]
No, there is no way to do this. No plans for it either at this time, unfortunately.
